### ISSUE TITLE
Allow searching for stock number (like Cxxxxxx)

### ISSUE THREAD

**Issue by m1geo** [Joeskaife/JLCQT#5]
Search doesn't respond if stock code number (Cxxxx) is put in the keyword search. Can be useful to search for stock codes, etc.

**Comment by Joeskaife** [comment]
I've added this but... it's a mixed-bag because strings that look like the LCSC part numbers occur all over the database to you get a lot of false matches.

**Comment by m1geo** [comment]
Do you search in the unique field for the LCSC stock code?

**Comment by Joeskaife** [comment]
No, the search currently looks in several columns for the string.

It may be that we need another search field or maybe I could search for text of the form "Cxxx" only in the parts column.... if I could be sure that form wouldn't be required in any other column. I guess the fact that you get so many matches with C1234 suggests that there are a lot of such strings in the description/manufacturer-part-number etc.
